src/compute_horde_facilitator_sdk/_internal/sdk.py

```diff
@@ -5,6 +5,7 @@
 import typing
 
 import httpx
+from compute_horde.executor_class import DEFAULT_EXECUTOR_CLASS, ExecutorClass  # type: ignore
```

Comment:
thats the type: ignore for here?

Reply:
long term solution  is of course adding py.typed marker in compute horde sdk (this will be fixed when that package is updated to  )

short term: add computehorde module to https://github.com/backend-developers-ltd/compute-horde-facilitator-sdk/blob/master/pyproject.toml#L143

Reply:
```
module = [
    "compute_horde.*",
 ```